### PR TITLE
Remove extra merchant info logging

### PR DESCRIPTION
**Pull request by elizalucas**
Removes logging merchant info when validation error occurs

### CODE REVIEW

File: server-middleware/apiApplePay.ts [hunk 1]
```diff
@@ -57,8 +57,6 @@ app.post('/validate', (req, res) => {
         res.send({
           errorMessage: a.message,
           responseStatus: a.response.status,
```
Comment (jcantwell) [inline server-middleware/apiApplePay.ts:59]: is the trailing comma here a problem or is this the standard for typescript?

Comment (elizalucas) [reply server-middleware/apiApplePay.ts:59]: standard for typescript

**Review by antiv0 — APPROVED** [review]

**Review by antiv0 — CHANGES_REQUESTED** [review]
you should remove error message also
it could have merchant info ?

**Review by antiv0 — APPROVED** [review]

**Comment by elizalucas** [comment]
> you should remove error message also it could have merchant info ?

This field is just "request failed with error <error #>"

**Review by jcantwell — APPROVED** [review]

**Pull request merged**